src/entities/offerings.ts

```diff
@@ -353,6 +354,8 @@ export interface Offering {
    * Weekly package type configured in the RevenueCat dashboard, if available.
    */
   readonly weekly: Package | null;
+
+  readonly paywall_components: PaywallData | null;
```

Comment:
Hmm I think it would be good to have this be camelCase `paywallComponents`. We should also mark this as experimental so devs don't use it accidentally.